lib/Controller/OAuthApiController.php

```diff
@@ -206,6 +206,7 @@ public function generateToken($grant_type, $code = null,
 		$refreshToken->setClientId($client->getId());
 		$refreshToken->setUserId($userId);
 		$refreshToken->setAccessTokenId($accessToken->getId());
+		$refreshToken->resetExpires();
```

Comment:
What is the rationale behind allowing RefreshToken setting expiration, and not Controller?